.github/workflows/demo-preview-destroy.yml

```diff
@@ -13,7 +13,7 @@ concurrency:
 
 jobs:
   destory:
-    if: ${{ github.event.pull_request.head.repo.full_name == 'primer/view_components' && github.actor != 'dependabot[bot]' && github.actor != 'primer-css' }}
+    if: ${{ github.event.pull_request.head.repo.full_name == 'primer/view_components' && github.actor != 'dependabot[bot]' }}
```

Comment:
realized this shouldn't be here or else we're never deleting release pr deployments